src/services/recipient.js

```diff
@@ -868,6 +876,8 @@ export async function getGoalsByActivityRecipient(
       collaborators: [],
       onAR: current.onAR,
       sessionObjectives: [],
+      responses: current.responses,
+      isFei: current.goalTemplateId === feiRootCauseFieldPrompt.goalTemplateId,
```

Comment:
why not have this defined in the call done on line 576, there is no need to have it done here. with the knowledge that there will soon be other standard goals, that may each have some custom UI work. I think we need to choose a better design on how to handle the special goals.

Reply:
Do you mean something like this: 

      [sequelize.literal(`"Goal"."goalTemplateId" = ${feiRootCauseFieldPrompt.goalTemplateId}`), 'isFei'],